docs/appendices/release-notes/unreleased.rst

```diff
@@ -63,6 +63,17 @@ Fixes
 .. stable branch. You can add a version label (`v/X.Y`) to the pull request for
 .. an automated mergify backport.
 
+- Fixed an issue that didn't allow to omit primary key columns which are
```

Comment:
Maybe we can avoid providing verbose example  - not sure about phrasing, smth like (roughly) 

"... caused PK/part by cols to be required on insert even if they have generated value based default not-null column"

Reply:
Thank you! I rephrased a bit, but I would keep the example, I think it doesn't harm to be verbose in the release notes.